platform/android/src/native_map_view.cpp

```diff
@@ -104,7 +104,9 @@ void NativeMapView::notifyMapChange(mbgl::MapChange change) {
 
     android::UniqueEnv _env = android::AttachEnv();
     static auto onMapChanged = javaClass.GetMethod<void (int)>(*_env, "onMapChanged");
-    javaPeer->Call(*_env, onMapChanged, (int) change);
+    if (javaPeer != NULL) {
```

Comment:
Could you make this just `if (javaPeer)` or `if (javaPeer != nulllptr)` if you want to make it more explicit